tests/app/services/test_query_builder.py

```diff
@@ -82,43 +84,42 @@ def test_should_make_multi_match_query_if_keywords_supplied(services_mapping):
     (build_query_params(keywords="something", filters={'lot': "lot"}), True),
     (build_query_params(filters={'serviceTypes': ["serviceTypes"]}), True)
 ))
-def test_should_identify_filter_search_from_query_params(query, expected):
-    return assert_equal, is_filtered(query), expected, query
+def test_should_identify_filter_search_from_query_params(services_mapping, query, expected):
+    return assert_equal, is_filtered(services_mapping, query), expected, query
 
 
-def test_should_have_filtered_root_element_if_service_types_search():
-    query = construct_query(build_query_params(filters={'serviceTypes': ["serviceTypes"]}))
+def test_should_have_filtered_root_element_if_service_types_search(services_mapping):
+    query = construct_query(services_mapping, build_query_params(filters={'serviceTypes': ["serviceTypes"]}))
     assert_equal("query" in query, True)
     assert_equal("bool" in query["query"], True)
     assert_equal("must" in query["query"]["bool"], True)
 
 
-def test_should_have_filtered_root_element_if_lot_search():
-    query = construct_query(build_query_params(filters={'lot': "SaaS"}))
+def test_should_have_filtered_root_element_if_lot_search(services_mapping):
+    query = construct_query(services_mapping, build_query_params(filters={'lot': "SaaS"}))
     assert_equal("query" in query, True)
     assert_equal("bool" in query["query"], True)
     assert_equal("must" in query["query"]["bool"], True)
 
 
-def test_should_have_filtered_root_element_and_match_all_if_no_keywords():
-    query = construct_query(build_query_params(filters={'serviceTypes': ["my serviceTypes"]}))
+def test_should_have_filtered_root_element_and_match_all_if_no_keywords(services_mapping):
+    query = construct_query(services_mapping, build_query_params(filters={'serviceTypes': ["my serviceTypes"]}))
     assert_equal("match_all" in query["query"]["bool"]["must"], True)
 
 
 def test_should_have_filtered_root_element_and_match_keywords(services_mapping):
-    query = construct_query(
-        build_query_params(keywords="some keywords",
-                           filters={'serviceTypes': ["my serviceTypes"]})
-    )["query"]["bool"]["must"]
+    query = construct_query(services_mapping, build_query_params(keywords="some keywords",
```

Comment:
I think I preferred the line breaks how they were, but neither option that pretty.